tasks/Task.cpp

```diff
@@ -60,8 +60,8 @@ void Task::updateHook()
 
 void Task::processIO()
 {
+    ControlCommand sentCommand = CONTROL_CMD_NO_COMMAND;
```

Comment:
Declare local variables close to where they are to be used (i.e. just before the `if(m_running)`)